### DISCUSSION TITLE
Attribute or Cloning (Day Note + 3 more dimensions)?

### DISCUSSION THREAD

**Discussion by vjz224** [TriliumNext/Trilium#3508] [Q&A] [answered]
Hello and thank you for taking the time to read my story. Standard disclaimer: new to Trilium didn't see any other discussions that pieced this together (I seriously tried). Feel free to point me to a thread that answers this in part or whole, I'm more than happy to do the research.

My objective: I want my notes to be searchable/filterable on multiple angles. In the same way Day Note works, but along other "dimensions". Basic background, I am a Program Manager in Professional Services. I host meetings with notes pertaining to one or more of the following dimensions at the same time:
- Date (already done)
- _Customer_
- _Product Teams_
- Individual attendees
My goal is easily lookup notes along any of these dimensions and see a historical view of notes regardless of where they are stored. My current tool and organization makes it easy to understand relevance to customer, that's how it's organized today. But what if I want to see context against the _product team_, regardless of _customer_. Or click on _someone_ who I've had meetings with and see all notes regardless of _customer_ or _product team_. I do these a lot and lose literal hours a week trying to piece these together. I need a better track record and lookup system.

I'm pretty sure Trilium can do it, but I'm not sure how to set it up.  
- Is this something that can / should be done with attributes and scripting (which I'll need to learn)? 
- Or is this a classic case for Note Cloning? (Pretty sure it won't work for linking)
- Combination of the two?

Better yet, if anyone is confident they can do it, I've got a donation to the project and a case of beer or 2 (Paypal or Venmo) if you replace OneNote for me and revolutionize how I organize.

**Comment by zadam** [comment, marked as answer]
Hi,

> Or is this a classic case for Note Cloning? (Pretty sure it won't work for linking)

Based on my shallow understanding of your needs - yes. The same note can appear in several different tree locations - under a date, customer, product team etc.

I'm not sure what you mean by linking, though, and why it wouldn't work with clones.

A bit similar concept you might want to take a look is a [saved search](https://github.com/zadam/trilium/wiki/Saved-search) in combination with adding labels to your notes.

**Reply by vjz224** [reply]
I'll take a look into saved search and confirm if I can make that work. Thank you!

**Reply by vjz224** [reply]
I got it to work more or less how I intended, saved search included. I just needed more time to play with the features.
Thank you!
